src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -166,7 +166,10 @@ export const PersonTableRow = ({
         dispatch(setNotification({ message: t('feedback.error.update_institution_user'), variant: 'error' }));
       }
     } else {
-      dispatch(setNotification({ message: t('feedback.error.update_person'), variant: 'error' }));
+      !cristinPerson.verified && isSuccessStatus(updateCristinPerson.status)
+        ? dispatch(setNotification({ message: t('feedback.success.update_person'), variant: 'success' })) &&
+          toggleDialog()
+        : dispatch(setNotification({ message: t('feedback.error.update_person'), variant: 'error' }));
```

Comment:
syns dette ble ganske tunglest, med kall på to funksjoner med `&&` innbakt i en ternary. kanskje bare bruke if-else i stedet?